### DISCUSSION TITLE
Can you run externals in Camomile plugins?

### DISCUSSION THREAD

**Discussion by unfamiliartunes** [pierreguillot/Camomile#262] [Q&A] [answered]
Title says it all. I recently downloaded some FFT externals and I wanted to use them in a plugin I'm building!

**Comment by eliottparis** [comment, marked as answer]
You may want to read [this thread](https://github.com/pierreguillot/Camomile/discussions/233)

<img width="906" alt="not_far_away" src="https://user-images.githubusercontent.com/1750257/117573576-ef59db00-b0d8-11eb-855d-1639fadee7f4.png">
